test/metabase/driver/postgres_test.clj

```diff
@@ -1028,7 +1028,7 @@
         (t2.with-temp/with-temp [Database database {:engine :postgres, :details test-user-details}]
           ;; make sure that sync still succeeds even tho some tables are not SELECTable.
           (binding [sync-util/*log-exceptions-and-continue?* false]
-            (is (some? (sync/sync-database! database))))
+            (is (some? (sync/sync-database! database {:scan :schema}))))
```

Comment:
Why?

Reply:
it's just faster for test that doesn't benefit from doing the analyze step.